Documentation/Profiling/davbr-blog-archive/README.md

```diff
@@ -0,0 +1,6 @@
+David Broman's Profiling Blog Archive
+===========================
+
+David Broman was the original author of many profiler features in the .NET 4/4.5 timeframe. At the time he had many helpful blog posts, and even today are some of the most thorough documentation we have available on how to use the profiling APIs. They are starting to show their age a bit, and talk about desktop .NET since they were written before .NET Core existed. However they are still largely accurate and a valuable source of documentation for the profiling APIs.
```

Comment:
The blog posts date back to 2005, which is around when desktop .NET 2.0 shipped.

Reply:
I didn't realize how old some of the posts were. The mention of which .NET version was not really relevant anyways, so I've changed it to just say he was an author of profiler features on desktop .NET